serenata_toolbox/federal_senate/dataset.py

```diff
@@ -25,9 +26,16 @@ def fetch(self):
             file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
             try:
                 urlretrieve(url, file_path)
-            except Exception as exception:
-                print('While fetching Seranata Toolbox not found file: {} \n{}'.format(file_path, exception))
-                not_found_files.append(file_path)
+            except HTTPError as http_error_exception:
+                print('We failed to reach the server')
+                print('Error code  ', http_error_exception.reason)
+                print('While fetching Seranata Toolbox not found file: {} \n{}'.format(file_path, http_error_exception))
```

Comment:
I would change the construction here to: `'While fetching, Serenata Toolbox didn't find the file: {} \n{}'`